src/xcode/ENA/ENA/Source/Scenes/RecycleBin/RecycleBinViewController.swift

```diff
@@ -257,7 +257,18 @@ class RecycleBinViewController: UITableViewController, FooterViewHandling {
 					self?.viewModel.restoreItem(at: indexPath)
 				}
 			)
-		case .coronaTest:
+		case .userCoronaTest:
```

Comment:
the alert in all three cases seems to be the same, why not one case then?

Reply:
There is a text difference between certificate and tests, but I didn't see a new text for family member tests, so combined the case now